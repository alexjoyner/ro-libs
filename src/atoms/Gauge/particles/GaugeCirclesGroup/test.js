import React from 'react';
import { shallow } from 'enzyme';
import { GaugeCirclesGroup } from './';

describe('GaugeCirclesGroup component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<GaugeCirclesGroup />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
