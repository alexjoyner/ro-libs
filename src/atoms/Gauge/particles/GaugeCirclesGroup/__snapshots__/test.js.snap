// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeCirclesGroup component Should match snapshot 1`] = `
<g>
  <ForwardRef
    cx="125"
    cy="150"
    fill="none"
    r="124"
  />
  <ForwardRef
    cx="125"
    cy="150"
    fill="none"
    id="gauge-range-boundary-circle"
    r="117"
  />
  <ForwardRef
    cx="125"
    cy="150"
    fill="none"
    id="gauge-inner-circle"
    r="100"
  />
</g>
`;
